docs/manual_tests/test_register.md

```diff
@@ -0,0 +1,112 @@
+  
+##### Test Description :   /register [method = POST] 
+- **Screenshot/gif** :
+
+![post-register-positive](https://user-images.githubusercontent.com/56113566/120222706-01edad00-c25e-11eb-96b0-d0a6d62534e8.gif)
+
+- **Expected Result** : A new user should be created.
+
+- **Actual Result** : Positive , User created successfully
+
+##### Test Description :   /register [method = POST] 
+- **Screenshot/gif** :
+
+![register-positive-1](https://user-images.githubusercontent.com/56113566/120224846-d7055800-c261-11eb-8989-2d8da8de84c1.png)
+
+- **Expected Result** : A new user should be created.
+
+- **Actual Result** : Positive , User created successfully
+
+![register-positive-1-res](https://user-images.githubusercontent.com/56113566/120224850-d8cf1b80-c261-11eb-8801-59f7fa4a3fa5.png)
+
+##### Test Description :   /register [method = POST] 
+- **Screenshot/gif** :
+
+
+![register-positive-2](https://user-images.githubusercontent.com/56113566/120225485-f8b30f00-c262-11eb-85d6-4805e2620372.png)
+
+- **Expected Result** : A new user should be created.
+
+- **Actual Result** : Positive , User created successfully
+
+![register-positive-1-res](https://user-images.githubusercontent.com/56113566/120224850-d8cf1b80-c261-11eb-8801-59f7fa4a3fa5.png)
+
+##### Test Description :   /register [method = POST] 
+- **Screenshot/gif** :
+
+![register-negative-6](https://user-images.githubusercontent.com/56113566/120224858-dc62a280-c261-11eb-84dd-7d1014b43b7e.png)
+
+- **Expected Result** : A new user shouldn't be created.
```

Comment:
This and the following tests can be put under the `Negative tests` section then give each tests subtitle based on the type of test being done, e.g. `If username already exist, etc`.